unstructured/partition/common.py

```diff
@@ -303,12 +302,12 @@ def convert_to_bytes(
     return f_bytes
 
 
-def convert_ms_office_table_to_text(table: docxtable.Table, as_html: bool = True):
+def convert_ms_office_table_to_text(table, as_html: bool = True):
```

Comment:
I think we can still include the typing by making the type a string, i.e.:
```suggestion
def convert_ms_office_table_to_text(table: "docxtable.Table", as_html: bool = True):
```

Reply:
Thanks!